cwms-data-api/src/main/java/cwms/cda/data/dao/BlobDao.java

```diff
@@ -142,7 +142,7 @@ public void create(Blob blob, boolean failIfExists, boolean ignoreNulls) {
         dsl.connection(c -> CWMS_TEXT_PACKAGE.call_STORE_BINARY(
                 getDslContext(c, blob.getOfficeId()).configuration(),
                 blob.getValue(),
-                blob.getId(),
+                blob.getId().toUpperCase(),
```

Comment:
Same as below comment. The casing can vary, only the comparison/lookup is case insensistive.